crates/turborepo-lib/src/shim.rs

```diff
@@ -41,6 +45,7 @@ static SUPPORTS_SKIP_INFER_SEMVER: &str = ">=1.7.0-canary.0";
 struct ShimArgs {
     cwd: PathBuf,
     skip_infer: bool,
+    verbosity: usize,
```

Comment:
I'm not sure if this is important, but we'll reparse the verbosity in `cli::run`: https://github.com/vercel/turbo/blob/main/crates/turborepo-lib/src/cli.rs#L106
I don't think these will drift, but if they do then the Go process could get a different verbosity level.

Reply:
Yea I did think about this, this is not likely to change too often which is the only reason I was ok with this - but we could parse the pre-parsed arg directly to the CLI and combine them